admin_guide/diagnostics_tool.adoc

```diff
@@ -116,37 +116,41 @@ current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
 
-[[additional-cluster-health-checks]]
-== Additional Diagnostic Checks via Ansible
-
-// TODO: add link to OCP image once it is available
-
-Some additional diagnostic checks are available through the *openshift-ansible*
-container image. See the image's link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[source repository] for usage information.
-
-The following health checks belong to a diagnostic task meant to be run against
-the Ansible inventory file for a deployed {product-title} cluster. They can
-report common problems for the current {product-title} installation.
+[[openshift-ansible-health-checks]]
+== openshift-ansible Health Checks
+
+Additional diagnostic health checks are available through *openshift-ansible*,
```

Comment:
We could use a conditional for the name, or in this particular case we could possible avoid giving it a name altogether:

> Additional diagnostic health checks are available through the Ansible-based tooling used to install and manage {product-title} clusters.